Index.html

```diff
@@ -1,122 +1,124 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
     <title>Document</title>
-</head>
-
-    <body>
-        <main id="main">
-            <h1 id="title">Dr. Norman Borlaug</h1>
-            <p>The man who saved a billion lives</p>
-            <p>and survived</p>
-            <figure id="img-div">
```

Comment:
You are referring to most of your elements in the CSS style sheet using Id's. Semantically, it is always advised to use classes as much as possible.

Reply:
Kindly phrase your comments in a polite and considerate manner, like a suggestion, instead of a command , you can use polite words such as PLease, kindly before you start your comment This will make it easy for the student to accept and implement them. :wink: